packages/bot-web-ui/src/pages/dashboard/announcements/announcements.tsx

```diff
@@ -0,0 +1,185 @@
+            });
+        };
+        modifyFieldDropdownValues('tradetypecat', 'accumulator');
+
+        const { derivWorkspace: workspace } = Blockly;
+
+        await load({
+            block_string: Blockly.Xml.domToText(strategy_dom),
+            file_name: 'Strategy with accumulator trade type',
+            workspace,
+            from: save_types.UNSAVED,
+            drop_event: null,
+            strategy_id: null,
+            showIncompatibleStrategyDialog: null,
+        });
+    };
+
+    const countActiveAnnouncements = (): number => {
+        return Object.values(amountAnnounce as Record<string, boolean>).reduce((count: number, value: boolean) => {
+            return value === true ? count + 1 : count;
+        }, 0);
+    };
+
+    const number_ammount_announce = countActiveAnnouncements();
+
+    const handleButtonClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+        event.stopPropagation();
+        setIsOpenAnnounceList(prevState => !prevState);
+    };
+
+    return (
+        <div className='announcements'>
+            <button
+                className='announcements__button'
+                onClick={handleButtonClick}
+                data-testid='btn-announcements'
+                ref={buttonRef}
+            >
+                <StandaloneBullhornRegularIcon fill='var(--icon-black-plus)' iconSize='sm' />
+                {!is_mobile && (
+                    <Text size='xs' line_height='s' className='announcements__label'>
+                        {localize('Announcements')}
+                    </Text>
+                )}
+                {number_ammount_announce !== 0 && (
+                    <div className='announcements__amount' data-testid='announcements__amount'>
+                        <p>{number_ammount_announce}</p>
+                    </div>
+                )}
+            </button>
+            <div className='notifications__wrapper' ref={wrapperRef}>
+                <Announcement
+                    className={clsx('', {
+                        'notifications__wrapper--mobile': is_mobile,
+                        'notifications__wrapper--desktop': !is_mobile,
+                    })}
+                    clearNotificationsCallback={() => {
+                        const announce_obj = Object.fromEntries(Object.keys(amountAnnounce).map(key => [key, false]));
+                        setAmountAnnounce(announce_obj);
+                    }}
+                    componentConfig={{
+                        clearButtonText: localize('Mark all as read'),
+                        modalTitle: localize('Announcement'),
+                        noNotificationsMessage: localize('No announcements MESSAGE'),
+                        noNotificationsTitle: localize('No announcements'),
+                    }}
+                    isOpen={isOpenAnnounceList}
+                    notifications={announcements}
+                    setIsOpen={() => handleButtonClick}
```

Comment:
@maryia-matskevich-deriv we don't need to do this, I figured it out yesterday

```suggestion
                    setIsOpen={() => {
                    // Need to be fixed from the UI library
                    }}
```